locale/en.po

```diff
@@ -2592,7 +2592,10 @@ msgstr ""
 msgid "EDITOR_TUTORIAL_REMOVE_ORGANELLE_TEXT"
 msgstr ""
 "Removing organelles also costs MP as it is a mutation to your species. You "
-"can right click organelles to remove them.\n"
+"can right click organelles to bring up the organelle menu.\n"
+"In the organelle menu you can select the delete option to \n"
```

Comment:
Just need a bit of tweaking:
`...can right click organelles to bring up the organelle menu and selecting the delete option to remove them.`

Reply:
Ok, tweaked the text as requested.